<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionGenerique</class>
 <widget class="QWidget" name="SectionGenerique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#windows {
background: gray;
}

#resumeConf{
color:black;
background-color: transparent;
border-width: 3px;
border-color: white;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 20px;
padding-left: 5px;
padding-right: 5px;
}

#mainFrame {
border: 3px solid gray;
border-radius: 40px;
border-image: url(:/img/Fond.jpg);
}

#subTitle{
background-color: transparent;
border-width: 3px;
border-color: white;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 15px;
padding-left: 5px;
padding-right: 5px;
}


QPushButton {
color: orange;
background-color: white;
border-width: 1px;
border-color: white;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 25px;
padding-left: 5px;
padding-right: 5px;
min-width: 280px;
max-width: 350px;
min-height: 25px;
max-height: 35px;
}

#groupBoxListe{
color:orange;
font-size: 18px;
}
#groupBoxOption{
color:orange;
font-size: 18px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="subTitle">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#5d5d5d;&quot;&gt;Nom Section&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="suivBouton">
     <property name="text">
      <string>Suivant</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="6">
    <widget class="QTextEdit" name="resumeConf">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>0</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxListe">
     <property name="title">
      <string>Listes Déroulantes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxOption">
     <property name="title">
      <string>Options à cocher</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
